UnityProject/Assets/Scripts/Player/JoinedViewer.cs

```diff
@@ -143,4 +172,24 @@ public void CmdRejoin(GameObject loggedOffPlayer)
 		SpawnHandler.TransferPlayer(connectionToClient, loggedOffPlayer, gameObject, EVENT.PlayerRejoined, null);
 		loggedOffPlayer.GetComponent<PlayerScript>().playerNetworkActions.ReenterBodyUpdates(loggedOffPlayer);
 	}
+
+	/// <summary>
+	/// Tells the client to start the countdown if it's already started
+	/// </summary>
+	[TargetRpc]
+	private void TargetSyncCountdown(NetworkConnection target, bool started, float countdownTime)
+	{
+		Logger.Log("Syncing countdown!", Category.Round);
+		UIManager.Instance.displayControl.preRoundWindow.GetComponent<GUI_PreRoundWindow>().SyncCountdown(started, countdownTime);
+	}
+
+	/// <summary>
+	/// Tells the server to start the round
+	/// </summary>
+	[Command]
+	public void CmdStartRound()
```

Comment:
This will need some validation, because otherwise anyone could trigger RoundStart() at any moment

Reply:
Removed it and added a server check in the window instead